app/v1/validation/abbreviatedReturn/AbbreviatedReturnValidator.scala

```diff
@@ -45,15 +45,6 @@ trait AbbreviatedReturnValidator extends BaseValidation {
     }
   }
 
-  private def validateAngie: ValidationResult[BigDecimal] = {
-    val angie: BigDecimal = abbreviatedReturnModel.angie.getOrElse(0)
-    angie match {
-      case a if a < 0 => NegativeAngieError(angie).invalidNec
-      case a if a % 0.01 != 0 =>AngieDecimalError(angie).invalidNec
```

Comment:
We can remove the case classes for these two errors now I believe?

Reply:
It's used for full return still